example/worker/merlin_worker.ml

```diff
@@ -1 +1,4 @@
+
 let () = Worker.run ()
+
+
```

Comment:
That's a lot of white space :-) 
We should ocamlformat this project, I will do that soon.

Reply:
oops, yes :-)